server/candlepin.spec.tmpl

```diff
@@ -139,6 +139,13 @@ touch %{buildroot}/%{_sysconfdir}/%{name}/%{name}.conf
 %{__install} -d -m 755 %{buildroot}/%{_sysconfdir}/tomcat/
 %{__unzip} %{SOURCE1} -d %{buildroot}/%{_sharedstatedir}/tomcat/webapps/%{name}
 
+## Add JSS to the classpath
+if grep -xq "^shared.loader=$" /etc/tomcat/catalina.properties ; then
+    sudo sed -i "/^shared.loader=$/ s/$/\/usr\/lib64\/jss\/jss4.jar/" /etc/tomcat/catalina.properties
```

Comment:
Unfortunately, this isn't going to work. I didn't do a good job explaining that the %install part in the spec file isn't actually run during the installation of the rpm, but when we build the rpm. Any commands to install files here, should be copying/updating files in the virtual build root directory structure. Editing the /etc/tomcat/catalina.properties file here is going to edit it in the machine we're building the rpm, not in the machine that the rpm is going to be installed. This is why other similar commands reference %{buildroot}, etc.

Reply:
Even if you dropped this in a `%post` section in the spec file, I don't think we want to go editing `catalina.properties` since that file belongs to the tomcat RPM and modifying a file owned by something else would be a no-no.

Instead, what if we dropped a file into `/etc/tomcat/conf.d` that looked something like this

```
CATALINA_OPTS="$CATALINA_OPTS -Dshared.loader=/usr/lib64/jss/jss4.jar"
```

This may not be perfect because I think this method would override any shared.loader defined in `catalina.properties`.  Also the `/etc/tomcat/conf.d` directory only existed as of four or five Fedora distributions ago, so that may not be an option in older RHEL 7 releases.  We'd need to check that the directory exists on the RHEL 7 versions we support (it is there on the RHEL 7.7 box I have handy).

Reply:
Removed this in favor of Tomcat's context.xml approach.